src/main/resources/lang/english.lang

```diff
@@ -1306,6 +1306,28 @@ heal reasons:
 	wither: withering, withered, wither potion, a wither effect, wither effect, wither
 	wither_spawn: wither spawning, wither spawn, wither summoning, a wither summoning, a wither spawn, a wither spawning
 
+# -- BossBar --
+# -- BossBar Color --
+bossbar colors:
+	blue: blue
+	green: green
+	pink: pink
+	purple: purple
+	red: red
+	white: white
+	yellow: yellow
```

Comment:
These can conflict with Skript's colour type, you should make sure this always works correctly
A possible fix for if it doesn't work, is to register a converter between the two classes (two converters, color -> barcolor and barcolor -> color), although I'm not sure if that'll fix everything
Another solution, if problems arise, is to remove the bar color type completely, and just switch to using the normal color type, and convert them to BarColor in expressions and stuff